### ISSUE TITLE
a11y: Cal-ITP slide deck template does not meet accessibility standards

### ISSUE THREAD

**Issue by leelas-pm** [cal-itp/calitp.org#493]
Review the presence of Cal-ITP slide deck PDFs on Cal-ITP websites and determine the best course of action (e.g. convert to web content, update deck template to meet a11y standards, etc.) and then create necessary design and dev tickets as needed.
